geode-core/src/test/java/org/apache/geode/management/internal/cli/functions/Bug3544JUnitTest.java

```diff
@@ -0,0 +1,182 @@
+    private long data;
+
+    public EmpProfile() {
+
+    }
+
+    public EmpProfile(long in_data) {
+      this.data = in_data;
+    }
+
+    public long getData() {
+      return data;
+    }
+
+    public void setData(long data) {
+      this.data = data;
+    }
+  }
+  public static class EmpData extends EmpProfile {
+    private short empId;
+    private Integer empNumber;
+    private long empAccount;
+
+    public EmpData() {
+      super();
+    }
+
+    public EmpData(long in_data, short in_empId, Integer in_empNumber, long in_empAccount) {
+      super(in_data);
+
+      this.empId = in_empId;
+      this.empNumber = in_empNumber;
+      this.empAccount = in_empAccount;
+
+    }
+
+    public boolean equals(Object other) {
+      if (this == other) {
+        return true;
+      }
+      if (other instanceof EmpData) {
+        return this.getEmpId() == (((EmpData) other).getEmpId());
+      }
+      return true;
+    }
+
+    @Override
+    public String toString() {
+      return "data:" + getData() + "," + "empId" + getEmpId();
+    }
+
+    public short getEmpId() {
+      return empId;
+    }
+
+    public void setEmpId(short empId) {
+      this.empId = empId;
+    }
+
+    public Integer getEmpNumber() {
+      return empNumber;
+    }
+
+    public void setEmpNumber(Integer empNumber) {
+      this.empNumber = empNumber;
+    }
+
+    public long getEmpAccount() {
+      return empAccount;
+    }
+
+    public void setEmpAccount(long empAccount) {
+      this.empAccount = empAccount;
+    }
+
+    @Override
+    public int hashCode() {
+      final int prime = 31;
+      int result = 1;
+      result = prime * result + (int) (empNumber ^ (empNumber >>> 32));
+      result = (int) (prime * result + empAccount);
+      result = prime * result + empId;
+      return result;
+    }
+  }
+
+  @BeforeClass
+  public static void setUp() throws Exception {
+    cache = new CacheFactory().set(MCAST_PORT, "0").create();
+    RegionFactory factory = cache.createRegionFactory(RegionShortcut.PARTITION);
+    region1 = factory.create(PARTITIONED_REGION);
+    EmpData emp_data_key = new EmpData(1, (short) 1, 1, 1);
+    region1.put(emp_data_key, "value_1");
+    ObjectMapper mapper = new ObjectMapper();
+    emp_key = mapper.writeValueAsString(emp_data_key);
+
+  }
+
+  @AfterClass
+  public static void tearDown() throws Exception {
+    cache.close();
+    cache = null;
+  }
+
+  /*
+   * This test addresses GEODE-3544
+   */
+  @Test
+  public void testLocateKeyIsObject() throws Exception {
+    DataCommandFunction dataCmdFn = new DataCommandFunction();
+
+    DataCommandResult result = dataCmdFn.locateEntry(emp_key, EmpData.class.getName(),
+        String.class.getName(), PARTITIONED_REGION, false);
+
+    assertNotNull(result);
+    result.aggregate(null);
+    List<DataCommandResult.KeyInfo> keyInfos = result.getLocateEntryLocations();
+    assertEquals(1, keyInfos.size());
+  }
+
+  @Test
```

Comment:
Is there supposed to be something different between these two tests?  They look to be identical.

Reply:
second test is removed now

Reply:
it was not required